lib/stdlib/doc/src/filename.xml

```diff
@@ -395,7 +395,7 @@ true
           case, the module must be known by the code server, that is,
           <c>code:which(<anno>Module</anno>)</c> must succeed.</p>
         <p><c><anno>Rules</anno></c> describes how the source directory can be
-          found when the object code directory is known. It is a list of
+          found when the module directory is known. It is a list of
```

Comment:
I am keeping this one as module because the first line of the doc already uses "module":

> Find the filename and compiler options for a module.